mautic/mautic
# Enabling Searching Contacts and Companies by Custom Field Values Without Field Alias Prefix

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 


## Description

Enabling Searching Contacts and Companies by Custom Field Values Without Field Alias Prefix

Problem Statement:
Currently, Mautic's search functionality only allows users to search for contacts or companies directly based on the values saved in custom fields, specifying the field alias and a colon (e.g., client_id:my-client-id).

---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull it down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create custom fields for contacts and companies by enabling the `Add to Search Index` flag
3. Create companies and contacts by adding data to the above-created custom field.
4. Search by the custom field values for Contact  
   1. Global search
   2. Contact listing page
5. Search by the custom field values for Contact  
    1. Company listing page
